hack/test-integration.sh

```diff
@@ -55,7 +55,7 @@ runTests() {
 trap cleanup EXIT
 
 # Convert the CSV to an array of API versions to test
-IFS=',' read -a apiVersions <<< ${KUBE_TEST_API_VERSIONS}
+IFS=',' read -a apiVersions <<< "${KUBE_TEST_API_VERSIONS}"
```

Comment:
I think this is what you want:

```shell
$ KUBE_TEST_API_VERSIONS='v1beta1,v1beta3'
$ apiVersions=(${KUBE_TEST_API_VERSIONS//,/ })
$ for apiVersion in "${apiVersions[@]}"; do echo "[$apiVersion]"; done
[v1beta1]
[v1beta3]
```

So this is transforming the comma-separated `KUBE_TEST_API_VERSIONS` into an `apiVersions` bash array, that you can then iterate on.

If this is a function, you should have `local` on all variables. (Doesn't seem to be the case here.)